stock_security_adj/views/ir_ui_menu_views.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.menu" id="stock.menu_warehouse_report">
+        <field name="groups_id" eval="[(6, 0, [ref('group_stock_operator')])]" />
+    </record>
+    <record model="ir.ui.menu" id="stock_account.menu_valuation">
```

Comment:
This inventory valuation menu opens product.product model in V11, but stock.valuation.layer model in V15.

The stock.valuation.layer model is only accessible to stock_manager in V15, so the operator should also have access to it.